pubspec.yaml

```diff
@@ -39,4 +40,5 @@ flutter:
   generate: true
 
   assets:
-    - assets/icons/
\ No newline at end of file
+    - assets/icons/
+    - assets/logo.png
```

Comment:
create a new folder called images and put into it the logo.

Then change this line to `- assets/images/`